### ISSUE TITLE
bug: Error on render when including options in markdown.lua file

### ISSUE THREAD

**Issue by moonlinx** [MeanderingProgrammer/markdown.nvim#53]
### Neovim version (nvim -v)

0.10.0

### Operating system

MacOS

### Terminal emulator / GUI

Wezterm

### Describe the bug

I updated to the latest version of your markdown.nvim in lazyvim and when I input the plugin like the lazy.nvim instructions, everything works fine. But the moment I add any the setups to change anything, it fails and hands me a rendering error. 

![issue@2x](https://github.com/MeanderingProgrammer/markdown.nvim/assets/49962728/83b88469-03f6-4781-9feb-ac6302c52ead)

### Expected behavior

Looking for the plugin to render the correct display once I exit out of insert mode

### Healthcheck output (:checkhealth render-markdown)

﻿﻿﻿
==============================================================================
render-markdown: require("render-markdown.health").check()

markdown.nvim [nvim-treesitter] ~
- OK installed
- OK markdown: parser installed
- OK markdown_inline: parser installed
- WARNING latex: parser not installed
  - ADVICE:
    - If you do not want LaTeX support avoid this warning by setting { latex_enabled = false }
- OK highlights enabled

markdown.nvim [executables] ~
- WARNING latex2text: not installed
  - ADVICE:
    - If you do not want LaTeX support avoid this warning by setting { latex_enabled = false }

markdown.nvim [configuration] ~
- ERROR dash: expected string, got table
- ERROR checkbox.checked: expected string, got table
- ERROR callout.caution: expected string, got table

**Comment by moonlinx** [comment]
I saw the breaking changes and fixed the issue. My apologies

**Comment by MeanderingProgrammer** [comment]
No worries, totally understandable!

Did you have any problems getting things working again? Just wanted to double check if the fixes made sense and worked or if I missed anything.

**Comment by moonlinx** [comment]
> No worries, totally understandable!
> 
> 
> 
> Did you have any problems getting things working again? Just wanted to double check if the fixes made sense and worked or if I missed anything.

Yes, the fixes made sense. Once I noticed and made the necessary changes then everything turned out fine. Seems that I was a bit impatient is all lol. Thank you for the follow up and for what you do with this plugin. It is amazing. I will continue to support however I can.

**Comment by MeanderingProgrammer** [comment]
Thank you for the kind words :)

Glad you are liking the plugin!
